frontend/src/components/pages/CreateReview/BookModal.tsx

```diff
@@ -0,0 +1,374 @@
+                    values={Object.values(BookFormats)}
+                    setSelectField={setFormat}
+                  />
+                  <AddStringInput
+                    id="isbn"
+                    label="ISBN"
+                    name="isbn"
+                    required
+                    inputField={isbn}
+                    setInputField={setIsbn}
+                    isInvalid={!isEmptyOrValidISBN()}
+                    errorMessage="Invalid ISBN format."
+                  />
+                  <FormControl id="price" isRequired width="45%">
+                    <FormLabel mb={2}>Price</FormLabel>
+                    <AddNumberInput
+                      mb={2}
+                      placeholder="$"
+                      setNumberField={setPrice}
+                      minNum={kMinPrice}
+                      maxNum={kMaxPrice}
+                    />
+                  </FormControl>
+                </Stack>
+              </Stack>
+            </GridItem>
+            <GridItem>
+              <Stack spacing={4} direction="column">
+                <AddStringInput
+                  id="bookCover"
+                  label="Book Cover"
+                  name="bookCover"
+                  placeholder="Image link here"
+                  required
+                  inputField={coverImage}
+                  setInputField={setCoverImage}
+                />
+                {/* <AddSelect
+                  id="genre"
+                  label="Genre"
+                  required
+                  values={Object.values(Genre)}
+                  setSelectField={setGenre}
+                /> */}
+                <FormControl
+                  isRequired
+                  width="100%"
+                  isInvalid={maxAge < minAge}
+                >
+                  <FormLabel mb={2}>Audience</FormLabel>
+                  <Stack direction="row" width="100%" spacing={4}>
+                    <AddNumberInput
+                      placeholder="Min Age"
+                      mb={0}
+                      setNumberField={setMinAge}
+                      minNum={kMinAge}
+                      maxNum={kMaxAge}
+                    />
+                    <AddNumberInput
+                      placeholder="Max Age"
+                      mb={0}
+                      setNumberField={setMaxAge}
+                      minNum={minAge}
+                      maxNum={kMaxAge}
+                    />
+                  </Stack>
+                  <FormErrorMessage>
+                    Max Age must be greater than or equal to Min Age{" "}
+                  </FormErrorMessage>
+                </FormControl>
+              </Stack>
+            </GridItem>
+          </Grid>
+        </ModalBody>
+        <ModalFooter pt={4}>
+          <Button
+            leftIcon={<CheckIcon />}
+            variant="add"
+            mr={4}
+            onClick={updateBookObj}
+            isDisabled={!hasRequired()}
```

Comment:
I think this prop should be `disabled` instead of isDisabled. Currently, the add button doesn't work as expected I think

Reply:
Oh oops good catch, that's b/c we ripped out genres and I forgot to remove that boolean check in the hasRequired() function. `isDisabled` seems to be the valid prop according to the [ChakraUI docs](https://chakra-ui.com/docs/components/form/button).